utils/Makefile

```diff
@@ -0,0 +1,15 @@
+ifneq ($(shell uname -m),aarch64)
+CROSS_COMPILE = aarch64-linux-gnu-
+endif
+
+CC = $(CROSS_COMPILE)gcc
+STRIP = $(CROSS_COMPILE)strip
+
+aarch64_32_bit_el1_supported: aarch64_32_bit_el1_supported.c
+	$(CC) -O2 -static -Wall -Wextra -Wpedantic -o $@ $^
```

Comment:
Setting `-Wpedantic` when gcc defaults to `-std=gnu17` is kind of funny.  Either set `-Wpedantic` AND use `-std=c17` (ie. an ISO C dialect, like c90, or c99, or c11) or set neither.

Reply:
Ha, good point, I added `-std=c17`.